ArduPlane/tailsitter.cpp

```diff
@@ -141,6 +141,14 @@ const AP_Param::GroupInfo Tailsitter::var_info[] = {
     // @Range: 10 500
     AP_GROUPINFO("RAT_VT", 17, Tailsitter, transition_rate_vtol, 50),
 
+    // @Param: THR_VT
+    // @DisplayName: Tailsitter forward flight to VTOL transition throttle
+    // @Description: Throttle used during FW->VTOL transition, -1 uses hover throttle
+    // @Units: percent
```

Comment:
```suggestion
    // @Units: %
```
this should get it past CI

Reply:
thanks...done....I removed the compensation becuase even if the thr was set for 15% it pushed it up to over 25% on a full battery....the non battery comp stuff must be messing with it...is there a batt comp only function to alter throttle without the expo and other curve adjustments?

Reply:
I think we actually have a bug here. For the assist case we are applying the linearization to throttle twice.